includes/commerce_pos.transaction.inc

```diff
@@ -492,7 +522,7 @@ function commerce_pos_transaction_form(&$form, &$form_state, $transaction_type)
 
   $form['parked_transactions'] = commerce_pos_parked_transactions_section($transaction_type, $wrapper_ajax, $has_active_transaction);
 
-  $form['#attached']['js'][] = drupal_get_path('module', 'commerce_pos') . '/js/commerce_pos.transaction.js';
+  $form['#attached']['js'][] = drupal_get_path('module', 'commerce_pos') . '/js/commerce_pos.js';
```

Comment:
Explanation in [this discussion](https://github.com/AcroMedia/commerce_pos/pull/28#discussion_r109998696).